lib/puppet/type/route53_spf_record.rb

```diff
@@ -0,0 +1,15 @@
+require_relative '../../puppet_x/puppetlabs/route53_record'
+
+Puppet::Type.newtype(:route53_spf_record) do
+  extend PuppetX::Puppetlabs::Route53Record
+  @doc = 'Type representing a Route53 SPF record.'
+  create_properties_and_params()
+
+  # SPF records should always be wrapped in double quotes
+  # this munge avoids the need to pass in a '"value"' to Puppet
+  values_property = self.properties.find { |item| item == Puppet::Type::Route53_spf_record::Values }
+  values_property.munge do |value|
+    value =~ /^".+"$/ ? value : "\"#{value}\""
```

Comment:
I understand @Iristyle's concerns, but I think that having it just do the right thing is really valuable. In my personal experience, a fairly large proportion of SPF records are set blindly via copy-paste from some vendor's docs rather than being properly understood.